src/components/TrackContributing.tsx

```diff
@@ -1,10 +1,225 @@
+import { Parser, Node } from 'commonmark'
 import React from 'react'
+import { useParams } from 'react-router-dom'
+
+import {
+  ConceptExerciseIssue,
+  useCreationConceptExerciseIssues,
+  useImproveConceptExerciseIssues,
+} from '../hooks/useConceptExerciseIssues'
+import { useRemoteConfig } from '../hooks/useRemoteConfig'
+import { LoadingIndicator } from './LoadingIndicator'
+import { PageLink } from './PageLink'
+import { TrackNewExerciseLocationState } from './TrackNewExercise'
+
+export interface TrackContributingParams {
+  trackId: TrackIdentifier
+}
 
 export function TrackContributing(): JSX.Element {
+  const { trackId } = useParams<TrackContributingParams>()
+  const { config, done } = useRemoteConfig(trackId)
+
   return (
     <div className="d-flex flex-wrap align-items-center mt-4 mb-4 row">
       <div className="col-12 mb-2">
-        <p>TODO: add contributing information</p>
+        {done ? <Content config={config} trackId={trackId} /> : <Loading />}
+      </div>
+    </div>
+  )
+}
+
+function Loading(): JSX.Element {
+  return (
+    <div style={{ position: 'relative', display: 'inline-block' }}>
+      <button type="button" style={{ background: 0, border: 0 }}>
+        <LoadingIndicator />
+      </button>
+    </div>
+  )
+}
+
+interface ContentProps {
+  trackId: TrackIdentifier
+  config: TrackConfiguration | undefined
+}
+
+function Content({ trackId, config }: ContentProps): JSX.Element {
+  const asyncOpenConceptExerciseIssues = useCreationConceptExerciseIssues(
```

Comment:
Looks like you could have used two components and directly destructure this too:

```suggestion
  const { done, result } = useCreationConceptExerciseIssues(
```

I also suggest using:

- `result`
- `loading`
- `error`

Instead of `done` + `result`.